mscore/inspector/inspector_slur.ui

```diff
@@ -41,7 +41,7 @@
       </font>
      </property>
      <property name="text">
-      <string>Slur/Tie</string>
+      <string></string>
```

Comment:
You can leave that string in, so GUI designers still see to, but mark it not to get translated, so translataors don't get bothered with it.
`<string notr="true">Slur/Tie</string>`